hera_cal/abscal.py

```diff
@@ -809,7 +809,8 @@ def abscal_arg_parser():
     a = argparse.ArgumentParser(description="command-line drive script for hera_cal.abscal module")
     a.add_argument("--data_files", type=str, nargs='*', help="list of file paths of data to-be-calibrated.", required=True)
     a.add_argument("--model_files", type=str, nargs='*', help="list of data-overlapping miriad files for visibility model.", required=True)
-    a.add_argument("--calfits_fname", type=str, default=None, help="name of output calfits file.")
+    a.add_argument("--calfits_files", type=str, nargs='*', help="list of calfits files to multiply with abscal solution before writing to disk.")
```

Comment:
I'd probably call this "--old_calfits_files" or "--calfits_infiles" or similar to clarify what this argument is.

(And then propagate the change below too).